src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -368,29 +377,56 @@ class DoujinDesu : ParsedHttpSource(), ConfigurableSource {
 
     override fun mangaDetailsParse(document: Document): SManga {
         val infoElement = document.select("section.metadata").first()!!
+        val AuthorName = when {
+            infoElement.select("td:contains(Author) ~ td")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> infoElement.select("td:contains(Author) ~ td")
+                .joinToString { it.text() }
+        }
+        val GroupName = when {
+            infoElement.select("td:contains(Group) ~ td")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> infoElement.select("td:contains(Group) ~ td")
+                .joinToString { it.text() }
+        }
+        val AuthorParser = when {
+            AuthorName.isNullOrEmpty() || AuthorName == "Tidak Diketahui" -> GroupName.takeIf { !it.isNullOrEmpty() && it != "Tidak Diketahui" } ?: "Tidak Diketahui"
+            else -> AuthorName
+        }
+        val CharacterName = when {
+            infoElement.select("td:contains(Character) ~ td")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> infoElement.select("td:contains(Character) ~ td")
+                .joinToString { it.text() }
+        }
+        val SeriesParser = if (infoElement.select("td:contains(Series) ~ td").text() == "Manhwa") {
+            infoElement.select("td:contains(Serialization) ~ td").text()
+        } else {
+            infoElement.select("td:contains(Series) ~ td").text()
+        }
+        val AlternativeTitle = when {
+            infoElement.select("h1.title > span.alter")
+                .isEmpty() -> "Tidak Diketahui"
+            else -> infoElement.select("h1.title > span.alter")
+                .joinToString { it.text() }
+        }
         val manga = SManga.create()
         manga.description = when {
-            document.select("section.metadata > div.pb-2 > p:nth-child(1)")
-                .isEmpty() -> "Tidak ada deskripsi yang tersedia bosque"
-            else -> document.select("section.metadata > div.pb-2 > p:nth-child(1)").first()!!.text()
+            infoElement.select("div.pb-2 > p:nth-child(1)")
+                .isEmpty() -> "Tidak ada deskripsi yang tersedia bosque\n\nJudul Alternatif : $AlternativeTitle\nGroup                 : $GroupName\nCharacter           : $CharacterName\nSeries                 : $SeriesParser"
+            else -> ((infoElement.selectFirst("div.pb-2 > p:nth-child(1)")?.text() ?: "") + "\n\nJudul Alternatif : $AlternativeTitle\nSeries                 : $SeriesParser")
         }
         val genres = mutableListOf<String>()
         infoElement.select("div.tags > a").forEach { element ->
             val genre = element.text()
             genres.add(genre)
         }
-        manga.author =
-            document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Author) + td:nth-child(2) > a")
-                .joinToString { it.text() }
+        manga.author = "$AuthorParser"
```

Comment:
```suggestion
        manga.author = AuthorParser
```

or

```suggestion
        manga.author = AuthorParser.toString()
```

Reply:
Got it 
Thanks for the suggestion 
Will commit that later